packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.module.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+
+import routing from './add.routing';
+import component from './add.component';
+import service from '../models.service';
+import capabilities from '../../../capabilities.service';
+
+import storageService from '../../../../storages/containers/containers.service';
+
+const moduleName = 'ovhManagerPciProjectServingNamespaceModelsAdd';
+
+angular.module(moduleName, [
+  'ngTranslateAsyncLoader',
+  'oui',
+  'ovh-api-services',
+  'pascalprecht.translate',
+  'ui.router',
+])
+  .config(routing)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .component('ovhManagerPciProjectServingNamespaceModelsAddComponent', component)
+  .service('OvhManagerPciServingModelsService', service)
+  .service('OvhManagerPciServingNamespaceModelsAddService', capabilities)
```

Comment:
I think it could more easy to understand what is the purpose of this service if it contains `capabilities` in its name.